packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -2319,7 +2319,91 @@ describe('DirectStripeRoutes', () => {
     });
   });
 
-  describe('updatePayment', () => {});
+  describe('updatePayment', () => {
+    let request;
+    const paymentToken = 'tok_visa';
+
+    beforeEach(() => {
+      request = request = { ...VALID_REQUEST, payload: { paymentToken } };
```

Comment:
Double assignment?

Reply:
aha good catch... that's embarrassing ':D